services/src/main/java/org/keycloak/services/util/ResolveRelative.java

```diff
@@ -29,13 +29,15 @@ public static String resolveRelativeUri(URI requestUri, String rootUrl, String u
         if (url == null || !url.startsWith("/")) return url;
         if (rootUrl != null) {
             return rootUrl + url;
-        } else {
+        } else if (requestUri != null) {
```

Comment:
This `if` condition is not need for KEYCLOAK-10807 but it should be added to avoid NPE.